lms/templates/_gated_content.html

```diff
@@ -0,0 +1,26 @@
+<%page args="prereq_url, prereq_section_name, gated_section_name" expression_filter="h"/>
+
+<%!
+from django.utils.translation import ugettext as _
+from openedx.core.djangolib.markup import Text
+%>
+
+<div class="xblock xblock-student_view xblock-student_view-vertical xblock-initialized">
+    <h2 class="hd hd-2 unit-title">
+        <span class="icon fa fa-lock" aria-hidden="true">&nbsp</span>${gated_section_name}
+    </h2>
+    <p/>
+    <h3 class="hd hd-3 problem-header">
+        ${_("Content Locked")}
+    </h3>
+    <p>
+        ${Text(_(
+            "You must complete the section '{prereq_section_name}' with the required score before you are able to view this content."
+        )).format(prereq_section_name=prereq_section_name)}
+        <p>
+            <a href="${prereq_url}" class="btn btn-brand">${_("Go to Prerequiste Section")}
+                <span class="sr">${_("Go to Prerequiste Section")}</span>
```

Comment:
No need for screen reader text here since it repeats the accessible text.  In this scenario, the screen reader hears the text repeated twice.